### PR TITLE
SVM: make sure to pop memcpy kwargs even with -O

### PR DESCRIPTION
**Pull request by matthiasdiener**
This fixes mirgecom SVM execution with -O

### CODE REVIEW

File: pyopencl/__init__.py [hunk 1]
```diff
@@ -1970,8 +1970,10 @@ def enqueue_copy(queue, dest, src, **kwargs):
 
         # These are NOT documented. They only support consistency with the
         # Buffer-based API for the sake of the Array.
-        assert kwargs.pop("src_offset", 0) == 0
-        assert kwargs.pop("dst_offset", 0) == 0
+        if kwargs.pop("src_offset", 0) != 0:
+            raise AssertionError
+        if kwargs.pop("dst_offset", 0) != 0:
+            raise AssertionError
```
Comment (inducer) [inline pyopencl/__init__.py]: It'd be better if (all of) these were `ValueError`s with an error message.

**Review by inducer — COMMENTED** [review]
Yikes, good catch! Thanks for the patch. Just one minor wrinkle. Once this is in, I'll do another release.

File: pyopencl/__init__.py (hunk 1 above)
Comment (matthiasdiener) [reply pyopencl/__init__.py]: What do you think of fd856a5?

Comment (inducer) [reply pyopencl/__init__.py]: That works, thanks!

**Comment by inducer** [comment]
https://pypi.org/project/pyopencl/2022.2.1/

**Pull request merged**